### ISSUE TITLE
first-time users do not go to their profile page to set given name and consent to receive communications.

### ISSUE THREAD

**Issue by DerekCaelin** [techmatters/terraso-web-client#1145]
## Description
First time users don't land in their profile screen

## Steps To Reproduce

1. Log in for the first time to Terraso
 
## Expected behavior
First-time user logs in

* direct to the profile page, where they set their given name and review communications preferences
* upon hit save, redirect to home (or relevant screen that they tried to view)

## Actual behavior
first-time user loads up home page

## Updated requirements
- [ ] as a user who selects "log in" and does not have an account, I should first be directed to the first time "profile" screen, then redirected to the page I was trying to join when I first tried to log in.

**Comment by Hanal-code** [comment]
**[Current]** 
Profile page - no given name error 
<img width="847" alt="image" src="https://github.com/techmatters/terraso-web-client/assets/92342777/647b41d4-6a52-40c3-b317-cc66df726ce3">

**Comment by paulschreiber** [comment]
Suggestion for the welcome text:
>Let us know what to call you, and what language you’d like to use.

The help text no longer unnecessary.

**Comment by DerekCaelin** [comment]
We met, we agreed #1 should be `Tell us what to call you and review your profile settings.`

**Comment by DerekCaelin** [comment]
We will address #3 in a separate issue.

**Comment by Hanal-code** [comment]
@josebui  I'm capturing what I reported in Slack regarding the first-time login screen flow: UC. The first-time user navigates to the Terraso app directly.
1. Terraso login 
2. Account Profile
3. Home screen 

UC. first-time user followed a login link from a public page in Terraso: (public) Story Map, (public) Group, etc.
1. (public) Story Map 
2. Terraso login 
3. Account Profile
4. Story Map main?

**Comment by Hanal-code** [comment]
UC. Unauthenticated user followed a link to an open group to join. 
1. (public) Group page
2. Terraso login  _--> take user to the group of interest_
3. (Member view) Group page

**Comment by Hanal-code** [comment]
@jose, I tested the following workflows with a new Terraso account (and new email addresses). 
1) first-time user followed a login link from a (public) Story Map. After login, landed on the Profile page! But after saving the given name, it didn't proceed to the Story map I started the workflow.
2) First-time followed a login link from a (public) group page. After login, landed on the Grouppage. The Profile page was skipped.
